examples/colab-notebook-processing_data_using_SQl_and_local_database/README.md

```diff
@@ -0,0 +1,3 @@
+# Colab Notebook for processing data using SQl and local database
+This folder contians the tutorial for <a href="https://github.com/vmware/versatile-data-kit/wiki/Processing-data-using-SQL-and-local-database"> Processing data using SQL and local database</a> example.
+Link to the Google Colab Notebook - <a href="https://colab.research.google.com/drive/1ujx6AaKsMkjeTeXvMUZQHwdFuLR0Qhz4?usp=sharing#scrollTo=CFrCNMbL80lX">Google Colab</a>
```

Comment:
The URL would be to github : 

`https://colab.research.google.com/github/vmware/versatile-data-kit/blob/main/<path-to-noteboook-file`